# Height of xy plane marker
svm_thre: -0.1
